Nightly builds sometimes fail with an ECONNRESET error

[Nightly builds](https://github.com/kraemer-lab/GRAPEVNE/blob/main/.github/workflows/nightly.yml) sometimes fail with an ECONNRESET error ([full log](https://github.com/kraemer-lab/GRAPEVNE/actions/runs/6400678573/job/17374624147)):
```
Run svenstaro/upload-release-action@v2
Error: Invalid response body while trying to fetch https://uploads.github.com/repos/kraemer-lab/GRAPEVNE/releases/122536604/assets?name=GRAPEVNE-0.2.1%20Setup.exe&: read ECONNRESET
```

The issue reports as a connection reset after the build process, during upload. The error appears intermittent. I have noticed this on the main repo and on my fork. The issue seems to only impact the `Setup.exe` upload when building for Windows - the nupkg and zip files upload as part of the same action without issue.

These issues on the github action repo appear relevant:
- https://github.com/svenstaro/upload-release-action/issues/ 62
- https://github.com/svenstaro/upload-release-action/issues/ 57

There appears to be a connection reset at some point, possibly because the windows build/upload is slower/longer (larger?) than the linux and macos builds, although the error is intermittent. This can probably be considered low priority for now as the zip and nupkg are available, new commits will force a nightly overwrite, and there are code signing issues #123 to deal with first, but the nightly build action cannot just be 're-run' since it first checks against the previous built commit hash to save on unnecessary rebuild time.

Addendum:

The macos nightly runner failed with a different network error - see here for the [full logs](https://github.com/jsbrittain/GRAPEVNE/actions/runs/6413632269/job/17412838497):
```
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.yarnpkg.com/@fortawesome/free-solid-svg-icons/-/free-solid-svg-icons-6.4.2.tgz: ESOCKETTIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/Users/runner/work/GRAPEVNE/GRAPEVNE/nodemapper/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

These appear to be network issues at github's end; possibly choose another time for the nightly build to avoid busy periods?

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -2,7 +2,7 @@ name: Nightly build
 on:
   workflow_dispatch:
   schedule:
-    - cron: "30 01 * * *"
+    - cron: "37 04 * * *"
 
 jobs:
   check:

